solr/core/src/test/org/apache/solr/search/TestCoordinatorRole.java

```diff
@@ -105,14 +102,11 @@ public void testSimple() throws Exception {
 
       assertEquals(10, rslt.getResults().size());
 
+      String SYNTHETIC_COLLECTION = CoordinatorHttpSolrCall.getSyntheticCollectionName("conf");
       DocCollection collection =
           cluster.getSolrClient().getClusterStateProvider().getCollection(SYNTHETIC_COLLECTION);
-      assertNotNull(collection);
-
-      Set<String> expectedNodes = new HashSet<>();
-      expectedNodes.add(coordinatorJetty.getNodeName());
-      collection.forEachReplica((s, replica) -> expectedNodes.remove(replica.getNodeName()));
-      assertTrue(expectedNodes.isEmpty());
+      // this should be empty as synthetic collection does not register with ZK
+      assertNull(collection);
```

Comment:
I think we should invoke the core API command to confirm a synthetic core is created

Reply:
Good call! And also we should confirm that there's no node created for the synthetic core.

Thank you for the suggestion!